net/openconnect/Makefile

```diff
@@ -8,12 +8,12 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=openconnect
-PKG_VERSION:=8.09
-PKG_RELEASE:=1
+PKG_VERSION:=8.10
+PKG_RELEASE:=2
```

Comment:
Shouldn't this be 1?

Reply:
You are right, i misread the tag definition!  Thank you!